docs/python/django/README.md

```diff
@@ -71,20 +71,20 @@ cd python/sqlcommenter-python && python3 setup.py install
 
 ### Enabling it
 
-Please edit your `settings.py` file to include `google.cloud.sqlcommenter.django.middleware.SqlCommenter` in your `MIDDLEWARE` section like this:
+Please edit your `settings.py` file to include `opentelemetry.opentelemetry.sqlcommenter.django.middleware.SqlCommenter` in your `MIDDLEWARE` section like this:
```

Comment:
type `opentelemetry.sqlcommenter` instead of `opentelemetry.opentelemetry.sqlcommenter`